linux_os/guide/system/accounts/accounts-session/accounts_users_netrc_file_permissions/ansible/shared.yml

```diff
@@ -0,0 +1,32 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- name: Get all local users from /etc/passwd
+  ansible.builtin.getent:
+    database: passwd
+    split: ':'
+
+- name: Create local_users variable from the getent output
+  ansible.builtin.set_fact:
+    local_users: '{{ ansible_facts.getent_passwd|dict2items }}'
+
+- name: Test for existence of .netrc file in home directories to avoid creating them, but only fixing permissions
+  ansible.builtin.stat:
+    path: '{{ item.value[4] }}/.netrc'
+  register: path_exists
+  loop: '{{ local_users }}'
+  when:
+    - item.value[2]|int >= {{{ gid_min }}}
+    - item.value[2]|int != 65534
```

Comment:
```suggestion
    - item.value[1]|int != 65534
```

Reply:
I believe the intention here is to identify interactive users regardless of their groups.

Reply:
Thanks for the suggestion :bow:  Well the point was to get the paths, to be honest can't remember, why I chose the `gid` criteria, over the `uid` one. Definitely makes sense that `uid` is a better criteria in this case.